docker-compose.yml

```diff
@@ -30,30 +30,13 @@ configs:
         options:
           socket: /tmp/docker.sock
       broker:
-        url: mqtt://broker:1883
+        url: mqtt://teamBroker:1883
         public_url: ws${TLS_ENABLED:+s}://mqtt.${DOMAIN:?error}
+        teamBroker:
+          enabled: ${TEAM_BROKER_ENABLED}
```

Comment:
If we use teamBroker as the default broker without a fallback to mosquito, do we still need this variable?

Reply:
Removed